proxy/src/main/java/net/md_5/bungee/connection/InitialHandler.java

```diff
@@ -668,4 +667,31 @@ public boolean isConnected()
     {
         return !ch.isClosed();
     }
+
+    public void relayMessage(PluginMessage input) throws Exception
+    {
+        if ( input.getTag().equals( "REGISTER" ) || input.getTag().equals( "minecraft:register" ) )
+        {
+            String content = new String( input.getData(), StandardCharsets.UTF_8 );
+
+            for ( String id : content.split( "\0" ) )
+            {
+                if ( registeredChannels.size() < 128 && id.length() < 128 )
```

Comment:
why are you checking the size of the set when you have code which is preventing it from going above 128?

Reply:
Right, I guess at this point I can just remove the BoundedHashSet implementation.

Reply:
no remove that check don't remove boundedhashset impl

Reply:
so it is checking only the id